src/Resources/views/Components/CookieBar.html.twig

```diff
@@ -1,59 +1,28 @@
-                            <div class="kmcc-collapsible-content__content js-kmcc-collapsible-content__content">
-                                {{ type.shortDescription | trans | replace_url | raw }}
-                            </div>
-                        </div>
-                    {% endif %}
-                </div>
-            {% endfor %}
-        </div>
-
-        <div class="kmcc-cookie-bar__footer">
-            <div class="kmcc-cookie-bar__footer__preferences">
-                {% set preferences %}
-                    <a class="js-kmcc-extended-modal-button" data-url="{{ path('kunstmaancookiebundle_legal_modal', {'internal_name': 'legal_cookie_preferences'}) }}">
-                        {{ 'kuma.cookie.bar.cookie_preferences' | trans }}
-                    </a>
-                {% endset %}
-                {% set privacy %}
-
-                    <a class="js-kmcc-extended-modal-button" data-target="legal_privacy_policy" data-url="{{ path('kunstmaancookiebundle_legal_modal', {'internal_name': 'legal_privacy_policy'}) }}">
-                        {{ 'kuma.cookie.bar.privacy_policy' | trans }}
-                    </a>
-                {% endset %}
-                {% spaceless %}
-                    <div class="kmcc-cookie-bar__footer__preferences__first">
-                        <p>{{ 'kuma.cookie.bar.footer' | trans({'%preferences%': preferences, '%privacy_policy%': privacy}) | raw }}</p>
-                    </div>
-                    <div class="kmcc-cookie-bar__footer__preferences__first--mobile">
-                        {# {{ preferences }} #}
-                        {{ privacy }}
-                    </div>
-                {% endspaceless %}
-
-                <div class="kmcc-cookie-bar__footer__preferences__second">
-
-                    {% include 'KunstmaanCookieBundle:Components:AcceptAllCookiesButton.html.twig' %}
-
-                    <a class="js-kmcc-extended-modal-button" data-target="legal_cookie_preferences" data-url="{{ path('kunstmaancookiebundle_legal_modal', {'internal_name': 'legal_cookie_preferences'}) }}">
-                        {{ 'kuma.cookie.bar.change_preferences' | trans | raw }}
-                    </a>
-                </div>
+
+    <div class="kmcc-cookie-bar__footer__preferences">
+        {% set preferences %}
+            <a class="js-kmcc-extended-modal-button" data-url="{{ path('kunstmaancookiebundle_legal_modal', {'internal_name': 'legal_cookie_preferences'}) }}">
+                {{ 'kuma.cookie.bar.cookie_preferences' | trans }}
+            </a>
+        {% endset %}
+        {% set privacy %}
+
+            <a class="js-kmcc-extended-modal-button" data-target="legal_privacy_policy" data-url="{{ path('kunstmaancookiebundle_legal_modal', {'internal_name': 'legal_privacy_policy'}) }}">
+                {{ 'kuma.cookie.bar.privacy_policy' | trans }}
+            </a>
+        {% endset %}
+
+
+
+        {% spaceless %}
```

Comment:
@sector32 I think it would make more sense to make the whole cookiebar spaceless and not just this little piece.